src/tools/illink/test/Mono.Linker.Tests.Cases/DataFlow/GenericParameterDataFlow.cs

```diff
@@ -432,7 +434,7 @@ public void PublicFieldsMethodLocalVariable ()
 				TypeRequiresPublicFields<TOuter> t = null;
 			}
 
-			[ExpectedWarning ("IL2091", nameof (TypeRequiresPublicMethods<TOuter>), ProducedBy = Tool.Trimmer | Tool.Analyzer)] // NativeAOT_StorageSpaceType
+			[ExpectedWarning ("IL2091", nameof (TypeRequiresPublicMethods<TOuter>), ProducedBy = Tool.Trimmer)]
```

Comment:
Could you please add a comment?
Keep the `NativeAOT_StorageSpaceType` and add something about the analyzer (that it doesn't handle local variables or something).